Formula/libpano.rb

```diff
@@ -20,20 +21,39 @@ class Libpano < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "4c9f6180154f15419ba4f2486eddd5f110098ebc3150b2007f867e9459c24b4b"
   end
 
+  depends_on "cmake" => :build
   depends_on "jpeg"
   depends_on "libpng"
   depends_on "libtiff"
 
-  # Fix -flat_namespace being used on Big Sur and later.
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-big_sur.diff"
-    sha256 "35acd6aebc19843f1a2b3a63e880baceb0f5278ab1ace661e57a502d9d78c93c"
-  end
+  uses_from_macos "zlib"
+
+  patch :DATA
```

Comment:
gonna submit this build patch

Reply:
looks like this is raised by alebcay in the upstream.
https://groups.google.com/u/1/g/Hugin-ptx/c/FZrsgbz9SNo